debian/bullseye/foreman/foreman-plugin.mk.in

```diff
@@ -1,16 +1,40 @@
 BUNDLE = @BUNDLE@
 
 build:
-	cp cache/* /usr/share/foreman/vendor/cache/
-	cp $(PLUGIN).rb /usr/share/foreman/bundler.d/
-	cd /usr/share/foreman && ( \
+	# prepare a local foreman installation based on packages
+	mkdir -p ./usr/share
+	cp -r /usr/share/foreman ./usr/share || true
```

Comment:
If the whole tree needs to be copied, it does make it a lot slower.

Just thinking out loud: you need the structure to exist and then add something to `bundler.d/` so the plugin exists, right? Could we in the foreman build package add something so you can use an env var to specify the entry and then use it as non-root?

You still may need to copy the vendor directory, or at least its cache. Possibly also NPM's node_modules.

Reply:
If by "a lot" you mean "one second", sure ;-)

```
[2022-10-17T08:23:06.186Z] # prepare a local foreman installation based on packages
[2022-10-17T08:23:06.186Z] mkdir -p ./usr/share
[2022-10-17T08:23:06.186Z] cp -r /usr/share/foreman ./usr/share || true
[2022-10-17T08:23:07.287Z] # clean up possibly existing state we copied
```

(from https://ci.theforeman.org/blue/organizations/jenkins/foreman-packaging-deb-pr-test/detail/foreman-packaging-deb-pr-test/813/pipeline/66 which ran with these changes)

And no, I need the whole code to be reachable, so that `bundle exec rake assets:precompile` works, which requires the app to be bootable.

Reply:
Hmm, I guess I was still thinking about non-NVME storage. Then this is easier.